src/timesync/timesyncd-manager.c

```diff
@@ -548,10 +549,18 @@ static int manager_receive_response(sd_event_source *source, int fd, uint32_t re
         receive = ntp_ts_to_d(&ntpmsg.recv_time);
         trans = ntp_ts_to_d(&ntpmsg.trans_time);
         dest = ts_to_d(recv_time) + OFFSET_1900_1970;
+        precision = exp2(ntpmsg.precision)
 
         offset = ((receive - origin) + (trans - dest)) / 2;
         delay = (dest - origin) - (trans - receive);
 
+        if (offset < -precision)
+            offset += precision;
+        else if (offset > precision)
+            offset -= precision;
+        else
+            offset = 0;
```

Comment:
Please use 8ch indentation.

Reply:
Sorry! I pushed this before double checking everything when you picked up the issue within seconds of me posting it. I wanded to show what my idea for a fix was so you could see what I was thinking.

Spacing is now fixed.